### ISSUE TITLE
Any way to get this working daily on its own? Bat file seems to break with scheduler.

### ISSUE THREAD

**Issue by Omniphantasm** [charlesbel/Microsoft-Rewards-Farmer#347]
I've tried task scheduler but it seems to break the script entirely when it runs the bat file.

If I run a bat that calls main.py it works perfectly fine, has blue text etc like it should.

If I use task scheduler to execute that exact same bat file it results in this where it hangs everytime and never proceeds.

←[38;5;39m2023-09-13 10:03:01,519 [INFO] Using default request storage←[0m
←[38;5;39m2023-09-13 10:03:01,525 [INFO] Created proxy listening on 127.0.0.1:53428←[0m
←[38;5;39m2023-09-13 10:03:01,526 [INFO] Using undetected_chromedriver←[0m
←[38;5;39m2023-09-13 10:03:02,392 [INFO] patching driver executable C:\Users\Omnif\appdata\roaming\undetected_chromedriver\undetected_chromedriver.exe←[0m

https://i.imgur.com/QJPOJN8.png

Anyone got a fix/solution/ELI5 how to get this running on its own daily?

Edit: Well hell, now it seems like its just hanging either way I start it so this might not even be right. I'll update again once I figure out whats going on -_-

**Comment by klept0** [comment]
Nuke the contents of your sessions folder then try it again - also make sure chrome is exiting fully when complete. 

If your problem is resolved - please close this issue out.

**Comment by Omniphantasm** [comment]
I have to either restart my windows session or just reboot and then it'll work if I run the bat file just fine.

Second it tries to run through task scheduler though same th ing happens and it hangs on the patching chrome part until next time I reboot.
